examples/gl/materialPBR/src/main.cpp

```diff
@@ -3,17 +3,21 @@
 
 //========================================================================
 int main( ){
-
+// PBR only works with the programmable renderer
+#ifdef TARGET_OPENGLES
```

Comment:
This might need to be emscripten as PI doesn't like gles 3 
Makes me think we should have some good system for fallbacks, like if you ask for 3 but it can't be provided fallback to 2 with a warning.

Reply:
Yeah agreed. Would be great if the default was the programmable renderer with the most recent GL version available for your targeted platform. I have yet to test, but iOS is OpenGL ES right? so it should/might work.

Reply:
Yeah definitely.  A  platform agnostic `windowSettings.glLatest = true;` would be super helpful. 
Probably also good to then talk about defaulting to programmable and having to request legacy GL if needed.